src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,78 @@
 import React from 'react';
+import { todoListType } from '../../types';
+import { Todo } from '../Todo';
 import './TodoList.scss';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+export class TodoList extends React.PureComponent {
+  state = {
+    search: '',
+    filteredTodos: 'all',
+  }
+
+  filter = {
+    all: () => true,
+    completed: todo => todo.completed,
+    active: todo => !todo.completed,
+  }
+
+  changeHandler = (event) => {
```

Comment:
```suggestion
  handleChange = (event) => {
```

use verbs
read https://github.com/kettanaito/naming-cheatsheet